evalml/tests/automl_tests/test_dask_engine.py

```diff
@@ -0,0 +1,215 @@
+import unittest
+
+import numpy as np
+import pytest
+import woodwork as ww
+from distributed import Client
+
+from evalml.automl.engine.dask_engine import DaskComputation, DaskEngine
+from evalml.automl.engine.engine_base import (
+    JobLogger,
+    evaluate_pipeline,
+    train_pipeline
+)
+from evalml.automl.engine.sequential_engine import SequentialEngine
+from evalml.pipelines.pipeline_base import PipelineBase
+from evalml.tests.automl_tests.dask_testing import (
+    TestBaselinePipeline,
+    TestLRCPipeline,
+    TestSVMPipeline,
+    automl_data
+)
+
+
+@pytest.mark.usefixtures("X_y_binary_cls")
+class TestDaskEngine(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.client = Client()
```

Comment:
Nice. This starts a local dask cluster, once for all of the tests in this class, correct? Any idea how many cores it'll take, how many workers it'll have?

Reply:
@dsherry instantiating with no arguments created a Client() based off a LocalCluster() with number of workers related to the number of available cores on the machine running it.